Tiltfile

```diff
@@ -147,3 +147,17 @@ def emoji():
   img = stop_build()
 
   return k8s_service(yaml, img)
+
+def words():
+  yaml = m4_yaml('words/deployments/words.yaml')
```

Comment:
For bonus points change this to use [kustomize](https://github.com/kubernetes-sigs/kustomize) and our new kustomize built-in? :P